.github/workflows/ci.yaml

```diff
@@ -13,8 +13,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp-version: [23.1.3]
-        elixir-version: [1.11.2]
+        otp-version: [23.1]
+        elixir-version: [1.11]
```

Comment:
👀

Reply:
Oups, _This is not the change you were looking for_ 👋🏻